client/components/App.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import axios from 'axios';
+import Image from './Image';
+import Carousel from './Carousel';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      images: [],
+      productId: window.location.pathname.split('/')[2],
+    };
+  }
+
+  componentDidMount() {
+    axios.get(`/images/${this.state.productId}`)
+      .then((pictures) => {
+        this.setState({
+          images: pictures.data,
+        });
+        return pictures.data;
```

Comment:
I don't think you need this return statement. Where are you returning it to?

Reply:
True. this was an experiment to try and make the tests work. I'll take it out.